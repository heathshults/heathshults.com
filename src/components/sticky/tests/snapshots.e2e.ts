import { snapIt } from '../../../test';

const component = 'sticky';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap(
      'sticky no scroll',
      `<hs-sticky top="20">
        <hs-alert open dismissible type="info">Sticky alert</hs-alert>
       </hs-sticky>`
    );
  });
});
