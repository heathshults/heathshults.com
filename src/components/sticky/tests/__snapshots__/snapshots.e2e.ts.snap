// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sticky renders sticky no scroll 1`] = `"<hs-sticky top=\\"20\\" class=\\"hydrated\\"><div><div><hs-alert open=\\"\\" dismissible=\\"\\" type=\\"info\\" class=\\"hydrated\\"><div role=\\"alert\\" class=\\"hs-alert hs-alert--info\\"><button class=\\"hs-button hs-button--close\\">Ã—</button>Sticky alert</div></hs-alert></div></div></hs-sticky>"`;
